arch/arc/Kconfig

```diff
@@ -105,6 +105,11 @@ endif #ISA_ARCV3
 config FP_FPU_DA
 	bool
 
+config ARC_DSP_COMPLEX
```

Comment:
Let's add help here with the information about what type of DSP HW this option implies.
Note that ARC has various DSP extensions and the generic name like `ARC_DSP_COMPLEX` doesn't help here :)